posts/2017-06-01-may.md

```diff
@@ -0,0 +1,75 @@
+
+We also updated the [example Eve applications](https://github.com/witheve/eve-starter/tree/master/programs) for v0.3. This entailed removing any references to databases, and updating records to use the new namespaced tags. We haven’t documented the various watchers yet (e.g. html, svg, events) so if you’re curious how to use these, you can use the example programs as a reference for now.
+
+### Documentation for v0.3
+
+
+As part of this release, we’re also making available our [documentation refresh](http://docs.witheve.com/v0.3/). We’ve updated the layout to match the new Eve website, and we’ve added some summary documents to help new users get oriented in the language. These include:
+
+- A new [quickstart](http://docs.witheve.com/v0.3/tutorials/quickstart.eve/) – We’ve got a new version of the quickstart that aims to be a 5 minute introduction to Eve. In 6 blocks we introduce blocks, records, and the difference between binding and committing records.
+
+- A new [syntax reference](http://docs.witheve.com/v0.3/syntaxreference/) – this is a 3 page document that acts as a reference for the Eve syntax. You can view it online, or print it out to keep handy on your desk.
+
+- A [library reference](http://docs.witheve.com/v0.3/handbook/libraries/) – This page will act as an index to the standard library. For every function in the standard library, we list its arguments as well as an example demonstrating its usage.
+
+These documents represent the minimum viable documentation to the Eve language. We hope you’ll take a look, and let us know if anything need more clarity or if we’ve omitted any important details. As we update the current docs for v0.3, they'll become available on the v0.3 docs site.
+
+One other note about documentation -- from now on we’ll be maintaining documentation for the current and future versions of Eve. To that end, when you go to [docs.witheve.com](http://docs.witheve.com), you’ll currently see an option to select v0.2 or v0.3
+
+### Community
+
+#### Eve Around the World
+
+**Copenhagen, June TBD**
+
+Just a reminder that Zubair is hosting a meetup in Copenhagen this June. We'll let you know when a date has been finalized.
+
+**San Francisco, Wednesday May 31**
+
+Last night we held our first official Eve meetup at our offices in downtown San Francisco. I'd like to thank everyone. Myself and the rest of the team enjoyed meeting everyone. We had an opportunity to finally meet some long-time users, as well as introduce ourselves to people finding Eve for the first time.
```

Comment:
regardless of how you do this chunk, "Myself" (dative) is grammatically incorrect; "I" (nominative) is correct